### PR TITLE
added use of FlipFlops

### PR DESCRIPTION
**Pull request by vinay72**
I have documented the  use of Flip Flops. #1

### CODE REVIEW

**Comment by vinay72** [comment]
@satu0king Hello, I am updated the file name to `flipflops.md` and also have styled the headings.

**Comment by vinay72** [comment]
@satu0king Don't consider my last commit which is on documentation of output elements. I am sorry for inconvenience.

File: Output Elements.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The <output> tag represents the result of a calculation.
+
+###Attributes
+
+
+   Attribute 	Value 	        Description
+   for 	        element_id 	Specifies the relationship between the result of the calculation, and the elements used in the calculation
+   form 	form_id 	Specifies one or more forms the output element belongs to
+   name 	name 	        Specifies a name for the output element
```
Comment (satu0king) [inline Output Elements.md]: What is this?

Comment (vinay72) [reply Output Elements.md]: @satu0king Okay , I understand now what you are trying to say.

**Pull request closed without merging**